extension/intents/nicknames/nicknames.js

```diff
@@ -114,3 +102,43 @@ intentRunner.registerIntent({
     intentRunner.unregisterPageName(name);
   },
 });
+
+intentRunner.registerIntent({
+  name: "nicknames.pause",
+  async run(context) {
+    if (context.parentRoutine === undefined) {
+      const exc = new Error("Command not available");
+      exc.displayMessage = "Command not available";
+      throw exc;
+    }
+    context.parentRoutine.stop = true;
+    browser.storage.sync.set({
+      pausedRoutine: {
+        name: context.parentRoutine.name,
+        nextIndex: context.parentRoutine.nextIndex,
+      },
+    });
+  },
+});
+
+intentRunner.registerIntent({
+  name: "nicknames.continue",
+  async run(context) {
+    const { pausedRoutine } = await browser.storage.sync.get("pausedRoutine");
+    if (pausedRoutine === undefined) {
+      const exc = new Error("Command not available");
+      exc.displayMessage = "Command not available";
```

Comment:
This should have a clearer message. Maybe like `continue can only be used in a routine after "pause routine"`